cmake/Hunter/config.cmake

```diff
@@ -83,21 +83,14 @@ if (APPLE AND (CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang" OR CMAKE_CXX_COMPILER
   )
 endif ()
 
-hunter_config(
-        wabt
-        URL https://github.com/qdrvm/wabt/archive/refs/tags/1.0.34-qdrvm1.zip
-        SHA1 d22995329c9283070f3a32d2c5e07f4d75c2fc31
-        KEEP_PACKAGE_SOURCES
-        CMAKE_ARGS
-        BUILD_TESTS=OFF
-        BUILD_TOOLS=OFF
-        BUILD_LIBWASM=OFF
-        USE_INTERNAL_SHA256=OFF
-)
-
 hunter_config(
     libsecp256k1
     VERSION 0.4.1-qdrvm1
     CMAKE_ARGS
         SECP256K1_ENABLE_MODULE_RECOVERY=ON
 )
+
+hunter_config(libp2p
+  URL  https://github.com/libp2p/cpp-libp2p/archive/e44b46754d3b58c257bb602d2ceac8b8e9c330e9.zip
```

Comment:
Please update to the tagged URL after https://github.com/libp2p/cpp-libp2p/pull/252 gets merged.